src/logic/safe/utils/safeVersion.ts

```diff
@@ -86,3 +88,17 @@ export const getSafeVersionInfo = async (safeVersion: string): Promise<SafeVersi
     logError(Errors._606, err.message)
   }
 }
+
+export const isValidMasterCopy = async (
+  chainInfo: ChainInfo,
+  safeAddress: string,
+  safeVersion: string,
+): Promise<boolean> => {
+  const masterCopyAddressFromProxy = await getMasterCopyAddressFromProxyAddress(safeAddress)
+  const { l2, chainId } = chainInfo
+  const useL2ContractVersion = l2 && semverSatisfies(safeVersion, '>=1.3.0')
```

Comment:
does the `semverSatisfies` deals successfully with `"1.3.0+L2"` like strings?

Reply:
Yes! I tried it.

Reply:
I added a test case for the version with meta info to ensure this works :)